### PR TITLE
Parser errors noted, but execution continues

### PR DESCRIPTION
**Pull request by slde-gorillaman**
This is a fix for #10 Output failing file on syntax error.

ParsedPhpFileFinder has been modified to collect PhpParser\Error and expose them to the outside.
CheckCommand can now detect if there were any parser errors and print them to the command line, but actual execution of the application continues

### CODE REVIEW

File: src/Finder/ParsedPhpFileFinder.php [hunk 1]
```diff
@@ -63,4 +67,20 @@ public function count()
     {
         return iterator_count(parent::getIterator());
     }
+
+    /**
+     * @return bool
+     */
+    public function hasParserErrors()
+    {
+        return (bool)count($this->parserErrors);
```
Comment (xabbuh) [inline src/Finder/ParsedPhpFileFinder.php]: I would either compare the return value of `count()` or use `empty()` (in Symfony we usually do the latter):

```php
return count($this->parserErrors) > 0;

// or

return !empty($this->parserErrors);
```

**Comment by MarvinKlemp** [comment]
ping @slde-gorillaman  :)

**Comment by slde-gorillaman** [comment]
pong @MarvinKlemp

File: src/Finder/ParsedPhpFileFinder.php (hunk 1 above)
Comment (slde-gorillaman) [inline src/Finder/ParsedPhpFileFinder.php]: Thanks for the hint. I corrected it

**Comment by xabbuh** [comment]
Do you really need `hasParserErrors()` at all? Letting `getParserErrors()` return an empty array seems perfectly valid to me so there wouldn't be the need to check for errors first.

**Comment by MarvinKlemp** [comment]
I'll merge this one now so that I can move tests and publish a new release

**Pull request merged**